### DISCUSSION TITLE
Not in this country rule

### DISCUSSION THREAD

**Discussion by cesare190** [mosparo/mosparo#208] [Q&A]
Is it possible to setup a rule that add spam point if the form is submitted everywhere expect a specific country?
i've found the country rule but i don't understand how can I invert the condition of a rule.

Thanks so much in advance

**Comment by zepich** [comment]
Hi @cesare190 

Thank you very much for your question.

No, it's not possible right now to invert a rule and not give all countries a number of points while one or two countries do not get points.

But I think it's a good suggestion to add. We could add an option 'All countries' for which you can set points and then define the countries you do not want to add points with a value of 0.

I will put your idea on the list of suggestions so we can add that feature in a future release.

If you want to make it harder for all countries except some, you can add very strict security settings in the general security settings. After that, you can add security policies and turn off the security settings for the countries you want or use less strict security settings.

Kind regards,

zepich

**Reply by zepich** [reply]
See #209

**Reply by cesare190** [reply]
Thanks!

**Reply by zepich** [reply]
Hi @cesare190 

With the newly released version 1.2, mosparo accepts now negative spam ratings.

You could use that for your use case. If you want to block all submissions from all countries except one, you can do the following:

1. Create a rule type IP address (Subtype: `Subnet`, value: `0.0.0.0/0`) or User Agent (Subtype: `Regex`, Value: `(.+)`). Use `5.0` as spam rating value
2. Create a rule type Country and add the countries you want. Use a negative value as a rating value, for example, `-5.0`

The logic is to catch all submissions with the first rule and give every submission many points so that it would be detected as spam. But for the ones from the countries you want to receive, you remove the points again with a negative rating value.

This is a simple fix in the tool that can help in your situation and other use cases. We will leave the idea with the 'all countries' option open since that is still a good idea for a future release.

Please let me know if you need any help.

Kind regards,

zepich
